styles/public/card-containers.scss

```diff
@@ -67,6 +67,7 @@
 			width: 100%;
 			padding: 0;
 			overflow: hidden;
+			-webkit-mask-image: -webkit-radial-gradient(white, black); // Safari fix
```

Comment:
Interesting issue. Is there any benefit to having it non-prefixed? ie: mask: radial-gradient(white, black);  Not that I think it'd be an issue beyond Safari but the coverage of non-prefixed gains about 7.5% usage according to caniuse.

Reply:
Yah, no harm in removing the prefix, I'll commit that change and will then Rebase and merge.